app/controllers/stories_controller.rb

```diff
@@ -369,7 +376,7 @@ def assign_listings
   def redirect_to_lowercase_username
     return unless params[:username] && params[:username]&.match?(/[[:upper:]]/)
 
-    redirect_to "/#{params[:username].downcase}", status: :moved_permanently
+    redirect_to "/#{params[:username].downcase}" + internal_nav_param, status: :moved_permanently
```

Comment:
Agreed, maybe the helper could have a comment explaining what `i=i` is and when to use it. Thanks!

ps. `"#{url}?i=i"` is more idiomatic and faster than `"#{url}" + "?i=i"